templates/post/single.html.twig

```diff
@@ -39,19 +39,19 @@
         {% include 'layout/_flash.html.twig' %}
 
         {% if user is defined and user and user.visibility is same as 'visible' and (user_settings.comment_reply_position == constant('App\\Controller\\User\\ThemeSettingsController::TOP')) %}
-        <section id="comment-add" class="section">
-            {% include 'post/comment/_form_comment.html.twig' %}
-        </section>
+            <div id="comment-add" class="section">
+                {% include 'post/comment/_form_comment.html.twig' %}
+            </div>
         {% endif %}
 
-        <section id="comments">
+        <div id="comments">
```

Comment:
I changed the DOM around in places to make post and entry match closer, removing differences where they didn't seem to make sense. I think the main difference at this point is when viewing a specific entry or post, the one being viewed is in a `<blockquote>` for post and `<article>` for entry, but DOM like comments below that are nearly synced up now, hoping to make styling easier/consistent and make it easier to unify these things